pkg/image/export/main.go

```diff
@@ -262,3 +278,110 @@ func findStrings(obj map[string]interface{}, found map[string]bool) {
 		}
 	}
 }
+
+func getImagesFromCharts(path string) ([]string, error) {
+	var images []string
+	imageMap := map[string]struct{}{}
+	chartVersion, err := getChartAndVersion(path)
+	if err != nil {
+		return nil, err
+	}
+	if err := filepath.Walk(path, func(p string, info os.FileInfo, err error) error {
+		return walkFunc(imageMap, chartVersion, path, p, info, err)
+	}); err != nil {
+		return images, err
+	}
+	for value := range imageMap {
+		images = append(images, value)
+	}
+	return images, nil
+}
+
+func getChartAndVersion(path string) (map[string]string, error) {
+	rtn := map[string]string{}
+	helm := libhelm.Helm{
+		LocalPath: path,
+		IconPath:  path,
+		Hash:      "",
+	}
+	index, err := helm.LoadIndex()
+	if err != nil {
+		return nil, err
+	}
+	for k, versions := range index.IndexFile.Entries {
+		var maxVersion *libhelm.ChartVersion
+		for _, version := range versions {
```

Comment:
I saw the `libhelm` has already done a reverse sorting for the `versions` array, can we use the `versions[0]` as the maximum version?